save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -461,6 +484,30 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
         }
     }
 
+    private fun setStatusAndNameAndSuiteAndTag(): String? {
+        val status1 = state.status?.let {
+            "&status=${state.status}"
+        } ?: run {
+            ""
+        }
```

Comment:
```suggestion
        } ?:  ""
```
I think `run {}` only overcomplicates code when there is a sinlge expression inside

Reply:
Done